tools/get_release_contributors.py

```diff
@@ -127,12 +134,12 @@ def get_user_by_email(self, key):
             return None
         email = m.group(1)
         # Handle Github-generated email addresses via static matching:
-        m = re.match(r'\A(\d+\+)?([^+@]+)\@users\.noreply\.github\.com\Z', email)
+        m = re.match(r'\A(\d+\+)?([^+@]+)@users\.noreply\.github\.com\Z', email)
```

Comment:
@ is not a special regex character (?), so this is should be ok?

Reply:
That's right, `@` is just any other character